docs/develop/user-input.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Get user input for scripts
+description: Add parameters to Office Scripts so users can control their experience. 
+ms.date: 08/15/2023
+ms.localizationpriority: medium
+---
+
+# Get user input for scripts
+
+Adding parameters to your script lets other users provide data for the script, without needing to edit code. When your script is run through the ribbon or a button, a prompt pops up that asks for input.
+
+:::image type="content" source="../images/user-input-example.png" alt-text="The dialog box shown to users when a script with parameters is run.":::
```

Comment:
In the image, you should replace your name with some test name

Reply:
Good catch! I removed that file location section for now (the UI might change and the "OneDrive" gets cutoff anyway)